wear/src/main/java/io/homeassistant/companion/android/home/views/SensorUi.kt

```diff
@@ -85,6 +86,11 @@ fun SensorUi(
                 overflow = TextOverflow.Ellipsis
             )
         },
+        secondaryLabel = {
+            sensor?.state?.let {
```

Comment:
To better match the phone sensors list
 - this should only be added if the sensor is enabled
 - this should skip the unit of measurement if the state is a string

(just [copy this](https://github.com/home-assistant/android/blob/6118933e4ef2eb8a20bc42f21f1dc25dd1371536/app/src/main/java/io/homeassistant/companion/android/settings/sensor/views/SensorListView.kt#L99-L111) except the enabled/disabled string which would be a bit much on a small screen)

Reply:
thanks added :)

Reply:
You inverted the check which does not work - if the state is a number it will show the unit of measurement, which is exactly what should be avoided.